VultisigApp/VultisigApp/Chains/thorchain.swift

```diff
@@ -8,7 +8,41 @@ import Tss
 import WalletCore
 
 enum THORChainHelper {
-    static let THORChainGas: UInt64 = 2000000
+    static let THORChainGas: UInt64 = getTHORChainGasPrice()
+    
+    private static var cachedTHORChainGas: UInt64?
+    
+    private static func fetchTHORChainGas(completion: @escaping (UInt64) -> Void) {
```

Comment:
This logic doesn't belongs to THORChainHelper , it should be in THORChainService, like all the other chains

Reply:
Yes, I am moving it to Thorchain Service.